src/views/Profile/components/ProfileAvatar.tsx

```diff
@@ -11,16 +11,14 @@ const TeamAvatar = styled.img`
   border: 1px solid ${({ theme }) => theme.card.background};
   border-radius: 50%;
   bottom: 0px;
-  height: 24px;
   position: absolute;
   right: 0px;
-  width: 24px;
+  width: 37.5%;
+  height: 37.5%;
```

Comment:
Why this value? Will this scale properly?

Reply:
Yea so I just checked the existing ratios between the smaller team image and the larger profile image, which are 

below mediaQueries.sm: `24px > 64px` - 37.5%
above mediaQueries.sm: `48px > 128px` - 37.5%

I'll also put a min-width of 20px on the team image?